pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+norecursedirs=dist build htmlcov docs .eggs
+addopts=-v -rxs --junitxml=junit-test-results.xml --cov=chaosk8s --cov-report term-missing:skip-covered --cov-report xml
```

Comment:
here, you should rename the project under coverage:

```
--cov=chaosext
```

The name is the name of the directory containing the source.